calculateMetrics/__init__.py

```diff
@@ -0,0 +1 @@
+"""Empty file required by pylint."""
```

Comment:
I don't think `_init_.py` is just to be there for pylint. Maybe do some research on what it actually does. It is a minor issue, though, since it's only a module docstring